.gitignore

```diff
@@ -0,0 +1,8 @@
+# .gitignore
+node_modules/
+
```

Comment:
- [ ] Kindly edit the Pull Request description as requested by the previous reviewer. You mentioned the changes you made to the title instead of the description. 
All the things you mentioned on the title should be here.
![Screenshot from 2022-11-29 19-10-47](https://user-images.githubusercontent.com/17243325/204584751-b3debb20-3156-4e35-930f-46c1416b183c.png)
Please refer [here](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md) to get more information about writing descriptive Pull Requests. ✍️